src/ProjectTemplates/Web.ProjectTemplates/content/EmptyWeb-CSharp/appsettings.Development.json

```diff
@@ -1,9 +1,9 @@
-{
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft": "Warning",
-      "Microsoft.Hosting.Lifetime": "Information"
-    }
-  }
-}
+{
```

Comment:
I'm not sure why these files are changing, but could you undo these?

Reply:
I reverted all whitespace changes.